src/wranglerjs/bundle.rs

```diff
@@ -42,6 +42,22 @@ impl Bundle {
             wasm_file.write_all(&wasm)?;
         }
 
+        script_file.write(
+            format!(
+                r#"
+                WebAssembly.instantiateStreaming =
```

Comment:
this is a neat trick! can we provide a real value for module or is it not possible?

Reply:
edit: didn't notice the format!, disregard

Reply:
sorry what do you mean by real value?